SharpBucket/V2/Pocos/Issue.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+
+namespace SharpBucket.V2.Pocos
+{
+    public class Issue
+    {
+        public int id { get; set; }
+
+        public IssuePriority priority { get; set; }
+
+        public IssueKind kind { get; set; }
+
+        public Repository repository { get; set; }
+
+        public IssueLinks links { get; set; }
+
+        public User reporter { get; set; }
```

Comment:
User is the only type which for legacy reasons is not suffixed `Info` but `Short` in its minimal form used in references.
But like for repository it should be the reduced type that should be used here
```suggestion
        public UserShort reporter { get; set; }
```